config/config.go

```diff
@@ -62,23 +60,28 @@ func FindAndReadConfig(fs afero.Fs, configFile string) (*v2.Config, error) {
 	if err != nil {
 		return nil, err
 	}
+
+	var c2 *v2.Config
 	switch v {
-	case 1:
+	case 1: //Upgrade the config version
+
 		c, err := v1.ReadConfig(b)
 		if err != nil {
 			return nil, err
 		}
 
-		c2, err := UpgradeConfigVersion(c)
-		return c2, err
-
+		c2, err = UpgradeConfigVersion(c)
```

Comment:
why change this code?

Reply:
I changed it to avoid duplication. The return statement occurred twice. Is that something I should consider changing?

Reply:
[this is just a style thing so opinions vary] I think that adding distance in the file between where the error/value is being generated and when it is being used adds for more complexity/makes it harder to read. 
I actually think the better solution would be to just `return UpgradeConfigVersion` or `return v2.ReadConfig` directly